SmartEVSE-3/data/index.html

```diff
@@ -788,7 +813,12 @@ <h1 id="version" class="h5 mb-0 text-white-800">Version: </h1>
                       window.location.href = "/settings";
                     }
                     function postPWM(value) {
-                       $.post("/modem?pwm=" + value);
+                       $.post("/settings?override_pwm=" + value);
```

Comment:
Please do not modify the current API; /settings is getting crowed so I want to reserve the /modem endpoint for all the modem/SoC stuff.....